### PR TITLE
Monitoring: fix error on Stackdriver exporter

### LINKED ISSUE #1330: Metrics: Export to Stackdriver is not working
There are no stackdriver metrics due to error in Labels:
```
Failed to export to Stackdriver: rpc error: code = InvalidArgument
 desc = One or more TimeSeries could not be written:
 Unrecognized resource label: instance_id: timeSeries[2,3,12];
 Unrecognized resource label: namespace_id: timeSeries[1,11]; 
Unrecognized resource label: pod_id: timeSeries[0,5,7]; 
Unrecognized resource label: zone: timeSeries[4,6,8-10]
```
**What happened**:

No stackdriver metrics on the dashboard, which was working several month ago.
New errors on Agones Controller logs.

**What you expected to happen**:
Stackdriver metrics are correctly visualised.

**How to reproduce it (as minimally and precisely as possible)**:
https://agones.dev/site/docs/guides/metrics/#stackdriver-installation

**What should be done to fix an issue**
- [x] Update documentation to use not `Global` but `Kubernetes Container` resource for each of gamesever, and upload new screenshot from Stackdriver.
- [x] Fix labels as in 
https://github.com/census-ecosystem/opencensus-go-exporter-stackdriver/issues/92
- [ ] Fix `view.Distribution` with zero range value.
- [ ] Optionally: update Stackdriver exporter library.

**Anything else we need to know?**:
There are two Pull requests which solved mentioned above ticket
They contain fixes for:
```
func getMonitoredResource(projectID string) (*monitoredres.MonitoredResource, error) {
...
}
```

**Environment**:
- Agones version: 
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration:
- Install method (yaml/helm):
- Troubleshooting guide log(s):
- Others:

### PR DESCRIPTION
**Pull request by aLekSer**
Zero value cannot be used as the only value for view.Distribution().

Closes #1330

### CODE REVIEW

**Comment by aLekSer** [comment]
Also found some useful examples. First uses new version of `client-go`, so we will need to update metrics with update to 1.15:
https://github.com/knative/pkg/blob/3c828cf99f67f8066e956490f634ebe90abd6892/metrics/reflector_test.go
Example of how distribution could be created:
https://github.com/knative/pkg/blob/4e57475bc87c1aba3e7fcc0207b593dc7186eb83/metrics/workqueue.go#L78

**Comment by aLekSer** [comment]
On Grafana Workerqueue metrics, I have changed looks the same as before.
Screenshot 2020-04-22 at 23 04 39
However I got a strange issue on `GameServers` dashboard, that could be caused by Grafana update:
```
Dashboard init (failed)
Failed create dashboard model
undefined is not iterable (cannot read property Symbol(Symbol.iterator))
```

**Comment by markmandel** [comment]
/assign @cyriltovena 

Can you review? I've no idea what any of this means :smile:

File: pkg/metrics/kubernetes_client.go [hunk 1]
```diff
@@ -81,14 +82,14 @@ func init() {
...
-		Aggregation: view.Distribution(0, 0.01, 0.025, 0.05, 0.075, 0.1, 0.25, 0.5, 0.75, 1, 2, 3),
+		Aggregation: view.Distribution(distributionSeconds...),
 	}))
 
 	runtime.Must(view.Register(&view.View{
 		Name:        "k8s_client_cache_list_items",
 		Measure:     cacheListItemCountStats,
 		Description: "Count of items in a list from the Kubernetes API.",
-		Aggregation: view.Distribution(0),
+		Aggregation: view.Distribution(0.00001),
```
Comment (cyriltovena) [inline pkg/metrics/kubernetes_client.go:92]: I wonder why past me didn't put any buckets here ?

I think we should have 0,10,50,100,150,250,300.

File: pkg/metrics/kubernetes_client.go [hunk 2]
```diff
@@ -109,14 +110,14 @@ func init() {
...
-		Aggregation: view.Distribution(0),
+		Aggregation: view.Distribution(distributionSeconds...),
 	}))
 
 	runtime.Must(view.Register(&view.View{
 		Name:        "k8s_client_cache_watch_events",
 		Measure:     cacheItemsInWatchesCountStats,
 		Description: "Number of items in watches on the Kubernetes API.",
-		Aggregation: view.Distribution(0),
+		Aggregation: view.Distribution(0.00001),
```
Comment (cyriltovena) [inline pkg/metrics/kubernetes_client.go:120]: Same here. you should use the same kind of buckets. It work for our dashboard to not have bucket cause I do average mostly but if you want 99th % you can't without proper buckets.

**Comment by cyriltovena** [comment]
Looks good just add bucket I forgot when I created this.

**Pull request merged**